src/app/authentication.guard.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { Observable } from 'rxjs/Observable';
+import { UserService } from './user.service';
+import { Router } from '@angular/router';
+
+@Injectable()
+export class AuthenticationGuard implements CanActivate {
+
+  constructor(private user:UserService, private router: Router) { }
```

Comment:
make sure you are using tslint (https://marketplace.visualstudio.com/items?itemName=eg2.tslint) code is not structured properly